.github/settings.yml

```diff
@@ -0,0 +1,41 @@
+repository:
```

Comment:
I don't think that there is a way to test this without merging to `develop`.

Reply:
Looked into this and came to the same conclusion.